### PR TITLE
Adding data to existing categories

### PR DESCRIPTION
**Pull request by NouranBakry**
I added data into the categories [book, city, country, color, music, electronic, language]

### CODE REVIEW

File: qual_id/categories/book.py [hunk 1]
```diff
@@ -1,21 +1,39 @@
...
+            'philosophy',
+            'crime',
+            'mystery',
+            'poetry',
+            'politics',
+            'fairytale',
+            'biography',
+            'autobiography',
+            'economics',
+            'business',
```
Comment (gabrielbarker) [inline qual_id/categories/book.py:32]: duplicate

File: qual_id/categories/instrument.py [hunk 2]
```diff
@@ -10,5 +10,9 @@ def get_values(self):
         'banjo',
         'cello',
         'violin',
-        'ukelele'
+        'ukelele',
+        'piano',
+        'organ',
+        'harp',
+        'cello'
```
Comment (gabrielbarker) [inline qual_id/categories/instrument.py:17]: duplicate

File: qual_id/categories/language.py [hunk 3]
```diff
@@ -1,23 +1,29 @@
...
+            'Tamil',
+            'Turkish',
+            'Korean',
+            'Italian',
+            'Thai',
+            'Persian',
+            'Filipino',
+            'Indonesian',
+            'Arabic',
+            'Persian',
```
Comment (gabrielbarker) [inline qual_id/categories/language.py:25]: duplicate

File: qual_id/categories/music.py [hunk 4]
```diff
@@ -18,4 +18,7 @@ def get_values(self):
             'kpop',
             'jpop',
             'punk',
+            'classical',
+            'hiphop',
+            'rock'
```
Comment (gabrielbarker) [inline qual_id/categories/music.py:23]: duplicate

**Review by gabrielbarker — CHANGES_REQUESTED** [review]
Hey @NouranBakry , great contribution, thanks!
Unfortunately I've found some duplicates. There may be more other than the ones I've found, but I've added comments. Please could you remove them?

**Comment by NouranBakry** [comment]
Of course I will remove them now. Sorry for the inconvenience.

File: qual_id/categories/country.py [hunk 5]
```diff
@@ -39,5 +39,7 @@ def get_values(self):
             'Moldova',
             'Hungary',
             'Belgium',
-            'Switzerland'
+            'Switzerland',
+            'Austria',
```
Comment (gabrielbarker) [inline qual_id/categories/country.py:43]: duplicate

**Review by gabrielbarker — CHANGES_REQUESTED** [review]

**Comment by gabrielbarker** [comment]
It's ok @NouranBakry 🙂  but there is still a failing test in the countries category. I've put a comment on but there could be others

**Review by gabrielbarker — APPROVED** [review]
Nice!

**Comment by gabrielbarker** [comment]
Thanks a lot for the contribution @NouranBakry ! You've went above and beyond  🙂  🎉

**Comment by NouranBakry** [comment]
Thank you so much for giving me the chance to make my first contribution 😊

**Pull request merged**